apps/emqx/src/emqx_schema.erl

```diff
@@ -968,26 +968,6 @@ fields("mqtt_wss_listener") ->
         ];
 fields("mqtt_quic_listener") ->
     [
-        {"certfile",
-            sc(
-                string(),
-                #{
-                    %% TODO: deprecated => {since, "5.1.0"}
-                    desc => ?DESC(fields_mqtt_quic_listener_certfile),
-                    importance => ?IMPORTANCE_HIDDEN
```

Comment:
just add `deprecated => {since,"5.1.0"}` prop is ok.
after uncommet this, the old config  check is ok, but with no certfile at running time  config.